### PR TITLE
your_first_game.rst: Clarify a section raised in an issue

### LINKED ISSUE #1625: Your First Game/Main Script Instructions: Confusing section
Original Text:
Next, click on the Player and connect the hit signal. We want to make a new function named game_over, which will handle what needs to happen when a game ends. Type “game_over” in the “Method In Node” box at the bottom of the “Connecting Signal” window. Add the following code, as well as a new_game function to set everything up for a new game:

Confusion:
click on the Player?
- Brand new to Godot, several things named Player. I was confused but ultimately sorted it. Perhaps adding the word Node will point people to what they need to click (especially since previous paragraph talks about dragging in scenes into Main).

Suggested Text edit:
Next, click on the Player Node and connect the hit signal in the Node editor. We want to make a new function named game_over, which will handle what needs to happen when a game ends. Type “game_over” in the “Method In Node” box at the bottom of the “Connecting Signal” window. Add the following code, as well as a new_game function to set everything up for a new game:

### PR DESCRIPTION
**Pull request by Meriipu**
Should close #1625

### CODE REVIEW

**Comment by Meriipu** [comment]
done editing now

File: getting_started/step_by_step/your_first_game.rst [hunk 1]
```diff
@@ -779,9 +779,13 @@ You can assign this property's value in two ways:
...
 
-Next, click on the Player and connect the ``hit`` signal. We want to make a
-new function named ``game_over``, which will handle what needs to happen when a
-game ends. Type "game_over" in the "Receiver Method" box at the bottom of the
+Next, click on the ``Player`` node and, if you have not, select the Node-tab
+instead of the Inspector-tab from the sidebar. You should see a list of the
+signals for the ``Player`` node. Find and double-click the ``hit`` signal in
+the list (or right-click it and select "Connect..."). This will open the
+"Connect a Signal" window. We want to make a new function named ``game_over``,
+which will handle what needs to happen when a game ends.
```
Comment (akien-mga) [inline getting_started/step_by_step/your_first_game.rst:787]: I'd rephrase a bit, e.g.:
```suggestion
Next, select the ``Player`` node in the Scene dock, and access the Node dock on
the sidebar. Make sure to have the Signals tab selected in the Node dock.

You should see a list of the signals for the ``Player`` node. Find and
double-click the ``hit`` signal in the list (or right-click it and select
"Connect..."). This will open the the signal connection dialog. We want to make
a new function named ``game_over``, which will handle what needs to happen when
a game ends.
```

Comment (Meriipu) [reply getting_started/step_by_step/your_first_game.rst]: skipped the double the and changed the second instance of `"Connect a Signal" window` into `signal connection dialog`

**Comment by akien-mga** [comment]
Thanks!
